development/contribute-to-docs.md

```diff
@@ -0,0 +1,147 @@
+Timescale documentation is hosted in a [GitHub repository][timescale-docs-github] 
+and is open for contribution from all community members. If you 
+find errors or would like to add content to our docs, this tutorial 
+will walk you through the process.
+
+# Pre-requisites
```

Comment:
These should be max, level `##` headers (probably `###` would be better)  Also, there is no hierarchy here?  Pre-requisites should probably be lower hierarchy than other steps.  'Making minor changes' is really a separate category from the other sections, which are all part of the same todo list